ci-operator/step-registry/gather/extra/gather-extra-commands.sh

```diff
@@ -40,6 +40,13 @@ oc --insecure-skip-tls-verify --request-timeout=5s get nodes -o jsonpath --templ
 oc --insecure-skip-tls-verify --request-timeout=5s get pods --all-namespaces --template '{{ range .items }}{{ $name := .metadata.name }}{{ $ns := .metadata.namespace }}{{ range .spec.containers }}-n {{ $ns }} {{ $name }} -c {{ .name }}{{ "\n" }}{{ end }}{{ range .spec.initContainers }}-n {{ $ns }} {{ $name }} -c {{ .name }}{{ "\n" }}{{ end }}{{ end }}' > /tmp/containers
 oc --insecure-skip-tls-verify --request-timeout=5s get pods -l openshift.io/component=api --all-namespaces --template '{{ range .items }}-n {{ .metadata.namespace }} {{ .metadata.name }}{{ "\n" }}{{ end }}' > /tmp/pods-api
 
+PLATFORM=$(oc get infrastructure cluster -o jsonpath="{.status.platform}")
+CAPI_PLATFORM=$(echo "$PLATFORM" | tr '[:upper:]' '[:lower:]')
+
+if [[ "${CAPI_PLATFORM}" == "baremetal" ]]; then
+  CAPI_PLATFORM=metal3
+fi
```

Comment:
We might need a special case for IBM Power VS too.

The crd prefix is `ibmpower` (see [here](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/bd1cb3fe6e519716c7d96e4535d058c91c0dd320/api/v1beta2/ibmpowervsmachine_types.go#L35)), whereas the `.status.platform` is `PowerVS` (see [here](https://github.com/openshift/api/blob/f14ed7c0a39529d00d64b6874b0a817b19aef98e/config/v1/types_infrastructure.go#L220-L221))

Reply:
Done